sdk/src/network-client.ts

```diff
@@ -715,7 +799,24 @@ class AleoNetworkClient {
     try {
       return await this.fetchData<Array<ConfirmedTransactionJSON>>("/block/" + height.toString() + "/transactions");
     } catch (error) {
-      throw new Error("Error fetching transactions. " + error);
+      throw new Error(`Error fetching transactions: ${error}`);
+    }
+  }
+
+  /**
+   * Returns the transactions present in the block with the specified hash.
+   * 
+   * @param {string} hash
+   * @example
+   * const transactions = networkClient.getTransactionsByHash("ab19dklwl9vp63zu3hwg57wyhvmqf92fx5g8x0t6dr72py8r87pxupqfne5t9");
+   */
+  async getTransactionsByHash(hash: string): Promise<Array<ConfirmedTransactionJSON>> {
```

Comment:
Same with method names.

```suggestion
  async getTransactionsByBlockHash(hash: string): Promise<Array<ConfirmedTransactionJSON>> {
```